mmhuman3d/utils/ffmpeg_utils.py

```diff
@@ -809,7 +814,8 @@ def images_to_gif(
         overwrite=True)
     num_frames = len(os.listdir(input_folder))
     start = (min(start, num_frames - 1) + num_frames) % num_frames
-    end = (min(end, num_frames - 1) + num_frames) % num_frames
+    end = (min(end, num_frames - 1) +
+           num_frames) % num_frames + 1 if end is not None else num_frames
```

Comment:
Similar as above.

Reply:
in ffmpeg, the `end_frame` must be positive